lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -1700,7 +1700,11 @@ void DWARFASTParserClang::GetUniqueTypeNameAndDeclaration(
   // For C++, we rely solely upon the one definition rule that says
   // only one thing can exist at a given decl context. We ignore the
   // file and line that things are declared on.
-  if (!die.IsValid() || !Language::LanguageIsCPlusPlus(language) ||
+  // For Rust, we do the same since Rust also has a similar qualified name?
+  // Is there a better way to do this for Rust?
```

Comment:
I would reword this to something like:
"FIXME: Rust pretends to be C++ for now, so use C++ name qualification rules"